pymorphy3/opencorpora_dict/wrapper.py

```diff
@@ -132,12 +131,10 @@ def iter_known_words(self, prefix=""):
         (default empty prefix means "all words").
         """
 
-        for word, (para_id, idx) in self.words.iteritems(prefix):
+        for word, (para_id, idx) in self.words.items(prefix):
```

Comment:
Here should be iteritems, see comment here: https://github.com/pymorphy2-fork/pymorphy2/blob/master/pymorphy2/opencorpora_dict/wrapper.py#L135

Reply:
Solved, and comments added for reference.